### PR TITLE
Reorg top nav3

### PR DESCRIPTION
**Pull request by smadbe**
Various improvement to the elements in the nav bars: redesign the lang picker, login/logout and user info on the top right.

### CODE REVIEW

File: src/app/core/components/language-picker/language-picker.component.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  width: 4.1rem;
+
+  &:hover {
+    background-color: $hover-mask;
+  }
+
+  .p-dropdown-label {
+    text-transform: uppercase;
+    color: white;
+  }
```
Comment (GabrielVidal1) [inline src/app/core/components/language-picker/language-picker.component.scss:15]: Only one letter is visible in the dropdown menu label, the two letters could be displayed by preventing the text ellipsis

image

**Review by GabrielVidal1 — CHANGES_REQUESTED** [review]
When connected as the usual user, clicking on 'Log out' does nothing and clicking on 'Profile' loads the profile page but throws an error (in the console)

File: src/app/core/components/language-picker/language-picker.component.scss (hunk 1 above)
Comment (smadbe) [reply src/app/core/components/language-picker/language-picker.component.scss:15]: I always have the 2 letters on my browser. Good idea, I'll prevent ellipses.

**Comment by smadbe** [comment]
> When connected as the usual user, clicking on 'Log out' does nothing and clicking on 'Profile' loads the profile page but throws an error (in the console)

For the log out not doing anything, that's because you have only changed the token, you should change the user_type to `authenticated` as well. (we could make a better use of the user_type vs user.is_temp, but that's more a improvement).

For the console error, that's something related with the `groups/me` page with the recently-changed menu, not related with this PR actually. I'll fix that in a separated PR.

**Review by GabrielVidal1 — APPROVED** [review]

**Pull request merged**